cql3/statements/cf_prop_defs.cc

```diff
@@ -129,11 +128,9 @@ void cf_prop_defs::validate(const db::extensions& exts) const {
         cp.validate();
     }
 
-    auto cdc_options = get_cdc_options();
-    if (cdc_options && !cdc_options->empty()) {
-        // Constructor throws if options are not valid
-        cdc::options opts(*cdc_options);
-    }
+    // This forces construction of cdc options, if they are present in extensions map
+    // Constructor throws if options are not valid
```

Comment:
I don't see a throw in get_cdc_options() below, what throws - to_options()?

Reply:
`cdc::options` have a constructor that throws when incorrect options are passed to it. The code above is no longer relevant in the new version, because the options are constructed in `cdc_extension`.